examples/next-ssr-app/pages/_app.tsx

```diff
@@ -15,7 +16,7 @@ function App({Component, pageProps}: AppProps): ReactElement {
                     content="Example of how to integrate Croct into React applications rendered on the client-side."
                 />
                 <title>Croct | React Next.js Example</title>
-                <script src="https://cdn.croct.io/js/v1/app/00000000-0000-0000-0000-000000000000/custom.js" />
+                <Script src="https://cdn.croct.io/js/v1/app/00000000-0000-0000-0000-000000000000/custom.js" />
```

Comment:
still wrong

Reply:
I thought that importing the component and replacing it would be right. I don't know what I would change in this part :(

Reply:
Missing the `strategy="beforeInteractive"`. Please build and run the application to ensure it'll work.